contracts/interfaces/IOnChainVault.sol

```diff
@@ -0,0 +1,91 @@
+// SPDX-License-Identifier: MIT
+
+pragma solidity ^0.8.19;
+
+import { IERC20 } from "@openzeppelin/contracts/token/ERC20/IERC20.sol";
+
+struct StrategyParams {
+    uint256 performanceFee;
+    uint256 activation;
+    uint256 debtRatio;
+    uint256 lastReport;
+    uint256 totalDebt;
+    uint256 totalGain;
+    uint256 totalLoss;
+}
+
+
+interface IOnChainVault {
+    error Vault__OnlyAuthorized(address); //0x1748142d
+    error Vault__V2(); //0xd30204e1
+    error Vault__V3(); //0xb22f5305
+    error Vault__V4(); //0x5f0c12c8
+    error Vault__NotEnoughShares(); //0x309d83b1
+    error Vault__ZeroToWithdraw(); //0xd498103e
+    error Vault__UnacceptableLoss(); //0x03fe7f1c
+    error Vault__InactiveStrategy(); //0x7ce4e353
+    error Vault__V6(); //0x6818be95
```

Comment:
What're those magical numbers in errors selectors? =)

Reply:
Error selectors, the will be replaced, but for now, there is no space for them